client/src/main/java/com/netflix/conductor/client/http/WorkflowClient.java

```diff
@@ -136,6 +164,31 @@ public String startWorkflow(String name, Integer version, String correlationId,
      */
     public String startWorkflow(StartWorkflowRequest startWorkflowRequest) {
```

Comment:
+javadoc
do you need all this logic wrapped with IOException or only a few lines?

Reply:
Added javadoc and the exception is thrown within the try-with-resources block.